File: Monstone_Android_Client/app/src/main/java/com/example/nightwingky/monstone/main_program/home/titleBar/TitleBarGridVO.java
package com.example.nightwingky.monstone.main_program.home.titleBar;

/**
 * Created by nightwingky on 17-3-18.
 */

public class TitleBarGridVO {

    private String title;
    private String introduction;
    private int imgSrc;

    public TitleBarGridVO() {
    }

    public TitleBarGridVO(String title, String introduction, int imgSrc) {
        this.title = title;
        this.introduction = introduction;
        this.imgSrc = imgSrc;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getIntroduction() {
        return introduction;
    }

    public void setIntroduction(String introduction) {
        this.introduction = introduction;
    }

    public int getImgSrc() {
        return imgSrc;
    }

    public void setImgSrc(int imgSrc) {
        this.imgSrc = imgSrc;
    }

    @Override
    public String toString() {
        return "TitleBarGridVO{" +
                "title='" + title + '\'' +
                ", introduction='" + introduction + '\'' +
                ", imgSrc=" + imgSrc +
                '}';
    }
}

File: kotlin_client/app/src/main/java/com/graduation/nightwingky/book_drifting/monstone/main_program/fragmentHome/titleBar/TitleBarGridVO.kt
package com.graduation.nightwingky.book_drifting.monstone.main_program.fragmentHome.titleBar

/**
 * Created by nightwingky on 17-3-18.
 */

class TitleBarGridVO {

    var title: String? = null
    var introduction: String? = null
    var imgSrc: Int = 0

    constructor() {}

    constructor(title: String, introduction: String, imgSrc: Int) {
        this.title = title
        this.introduction = introduction
        this.imgSrc = imgSrc
    }

    override fun toString(): String {
        return "TitleBarGridVO{" +
                "title='" + title + '\''.toString() +
                ", introduction='" + introduction + '\''.toString() +
                ", imgSrc=" + imgSrc +
                '}'.toString()
    }
}
